Dockerfile

```diff
@@ -1,12 +1,25 @@
-FROM golang:1.15.3-buster AS builder
-RUN apt-get update -y
-RUN apt-get install -y libgpgme-dev
-WORKDIR /go/src/tagger
+#
+# Builder
+#
+
+FROM registry.fedoraproject.org/fedora:latest AS builder
+
+WORKDIR /src
 COPY . .
-RUN go build -o /tagger ./cmd/tagger
 
+RUN yum install -y \
+    golang \
+    gpgme-devel \
+    make
+
+RUN make
+
+#
+# Application
+#
+
+FROM registry.fedoraproject.org/fedora:latest
```

Comment:
Here also I think we should use one of the UBI images, would that be feasible? I don't know even if those are public.